.circleci/config.yml

```
@@ -80,18 +81,19 @@ jobs:
 
     steps:
       - checkout
-      - run:
-          name: Install System Dependencies
-          command: sudo apt-get update && sudo apt-get install -y libsndfile1
+   #    - run:
+   #        name: Install System Dependencies
+   #        command: sudo apt-get update && sudo apt-get install -y libsndfile1 texlive-latex-extra dvipng
```

- run:
          name: Install System Dependencies
          command: sudo apt-get update && sudo apt-get install -y libsndfile1
```suggestion
   #    - run:
   #        name: Install System Dependencies
   #        command: sudo apt-get update && sudo apt-get install -y libsndfile1 texlive-latex-extra dvipng
```